### PR TITLE
LPS-102602 Service Builder Implementation

### PR DESCRIPTION
**Pull request by cgoncas**

### CODE REVIEW

**Comment by brianchandotcom** [comment]
Hey @cgoncas this looks pretty good.

Can you make a slight change:

		<column name="canonicalURLEnabled" type="boolean" />
		<column localized="true" name="canonicalURL" type="String" />
		<column name="openGraphTitleEnabled" type="boolean" />
		<column localized="true" name="openGraphTitle" type="String" />
		<column name="openGraphDescriptionEnabled" type="boolean" />
		<column localized="true" name="openGraphDescription" type="String" />
		<column name="openGraphImageFileEntryId" type="long" />

make it

		<column localized="true" name="canonicalURL" type="String" />
		<column name="canonicalURLEnabled" type="boolean" />
		<column localized="true" name="openGraphDescription" type="String" />
		<column name="openGraphDescriptionEnabled" type="boolean" />
		<column localized="true" name="openGraphTitle" type="String" />
		<column name="openGraphTitleEnabled" type="boolean" />
		<column name="openGraphImageFileEntryId" type="long" />

Basically, everything between than private/layoutId, and lastPublishDate - I'm probably going to just sort.

Thx!

**Comment by cgoncas** [comment]
Continues in: https://github.com/brianchandotcom/liferay-portal/pull/79393
